extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/runtime/Codecs.java

```diff
@@ -0,0 +1,176 @@
+package io.quarkus.websockets.next.runtime;
+
+import java.lang.reflect.Type;
+import java.util.List;
+
+import jakarta.enterprise.inject.Any;
+import jakarta.enterprise.inject.Instance;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.arc.All;
+import io.quarkus.websockets.next.BinaryMessageCodec;
+import io.quarkus.websockets.next.MessageCodec;
+import io.quarkus.websockets.next.TextMessageCodec;
+import io.vertx.core.buffer.Buffer;
+
+@Singleton
+public class Codecs {
+
+    private static final Logger LOG = Logger.getLogger(Codecs.class);
+
+    @Inject
+    @Any
+    Instance<TextMessageCodec<?>> textCodec;
```

Comment:
Is this even used? I can't find any usage (but I'm not searching very thoroughly) and it seems weird that there's nothing like this for binary codecs.

Reply:
It is not. Just a leftover...